lib/classes/CLI.js

```diff
@@ -83,6 +84,14 @@ class CLI {
     }
   }
 
+  isConfigIndependentRequest(processedInput) {
+    const commands = processedInput.commands.join(' ');
+
+    if (configIndependentCommands.has(commands)) return true;
+
+    return false;
```

Comment:
Let's write it as `return configIndependentCommands.has(processedInput.commands.join(' '))`